packages/core/src/external/aws/location.ts

```diff
@@ -0,0 +1,33 @@
+import * as AWS from "aws-sdk";
+import { PromiseResult } from "aws-sdk/lib/request";
+import dayjs from "dayjs";
+import duration from "dayjs/plugin/duration";
+dayjs.extend(duration);
+
+export function makeLocationClient(region: string): AWS.Location {
+  return new AWS.Location({ signatureVersion: "v4", region });
+}
+
+export function getLocationResultPayload({
+  result,
+}: {
+  result: PromiseResult<AWS.Location.SearchPlaceIndexForTextResponse, AWS.AWSError>;
+}) {
+  // TODO: copy lambda.ts structure here
+  return result.Results;
+}
+
+export function getCoordinatesFromLocation({
+  result,
+}: {
+  result: PromiseResult<AWS.Location.SearchPlaceIndexForTextResponse, AWS.AWSError>;
+}) {
+  const resp = getLocationResultPayload({ result });
+  const topSuggestion = resp[0]?.Place?.Geometry?.Point;
+
+  if (topSuggestion && topSuggestion[0] && topSuggestion[1]) {
+    return { lon: topSuggestion[0], lat: topSuggestion[1] };
```

Comment:
Not a huge deal, especially if still adjusting this, but it's better to store array items on const and work off of that than referring to array items